source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,127 @@
+.. _kotlin-sync-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+The Atlas Search feature enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. The indexes specify the behavior of
+the search and which fields to index.
+
+You can call the following methods on a collection to manage your Atlas Search
+indexes:
+
+- ``createSearchIndex()``
+- ``createSearchIndexes()``
+- ``listSearchIndexes()``
+- ``updateSearchIndex()``
+- ``dropSearchIndex()``
+
+.. note::
+
+   The Atlas Search Index management methods run asynchronously. The
```

Comment:
I don't think "index" should be capitalized since it's not part of the product name
```suggestion
   The Atlas Search index management methods run asynchronously. The
```